telegram/_bot.py

```diff
@@ -715,9 +715,12 @@ async def send_message(
                 :tg-const:`telegram.constants.MessageLimit.MAX_TEXT_LENGTH` characters after
                 entities parsing.
             parse_mode (:obj:`str`): |parse_mode|
-            entities (List[:class:`telegram.MessageEntity`], optional): List of special entities
-                that appear in message text, which can be specified instead of
+            entities (Sequence[:class:`telegram.MessageEntity`], optional): Sequence of special
+                entities that appear in message text, which can be specified instead of
                 :paramref:`parse_mode`.
+
+                .. versionchanged:: 20.0
+                    Accept :obj:`Sequence` instead of :obj:`List`.
```

Comment:
let's make this an rst substitution (see docs/substituions/global.rst), we could do:

```
.. |sequenceargs| replace:: Accepts any :class:`collections.abc.Sequence` as input instead of just a list.
``` 
then `sequenceclassargs` would become:
```
.. |sequenceclassargs| replace:: |sequenceargs| The input is converted to a tuple.
```